packages/flutter/test/widgets/scrollable_fling_test.dart

```diff
@@ -44,6 +44,11 @@ void main() {
     expect(getCurrentOffset(), dragOffset);
     await tester.pump(const Duration(seconds: 5));
     final double androidResult = getCurrentOffset();
+    // Regression test for https://github.com/flutter/flutter/issues/83632
+    // Before changing these values, ensure the fling results in a distance that
+    // makes sense. See issue for more context.
+    expect(androidResult, greaterThan(394.0));
+    expect(androidResult, lessThan(395.0));
```

Comment:
I added this to draw attention to any future regressions. The rest of the PR is a straight revert from the original.